index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Nova</title>
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/base.css" />
+    <link rel="stylesheet" href="./styles/header.css" />
+    <link rel="stylesheet" href="./styles/hero.css" />
+    <link rel="stylesheet" href="./styles/features.css" />
+    <link rel="stylesheet" href="./styles/section-white.css" />
+    <link rel="stylesheet" href="./styles/pricing.css" />
+    <link rel="stylesheet" href="./styles/team.css" />
+    <link rel="stylesheet" href="./styles/footer.css" />
+    <script
+      src="https://kit.fontawesome.com/050cb77415.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="hero--background">
+      <header class="section">
+        <div class="header__container container row">
+          <div class="logo header__logo">
+            <img class="logo__image" src="./images/logo.svg" alt="nova logo" />
+          </div>
+          <nav class="nav header__nav">
+            <ul class="nav__links row">
+              <li class="nav__link"><a href="#team">the team</a></li>
+              <li class="nav__link"><a href="#pricing">pricing</a></li>
+              <li class="nav__link"><a href="#features">features</a></li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+
+      <section class="section section-hero">
+        <div class="hero container row">
```

Comment:
Tutaj jest pewna niekonsekwencja. Skoro tutaj zaczynamy block `hero` to w linii 23 nie powinien występować jego modyfikator. Modyfikator nie może "wychodzić" poza obszar bloku/elementu.